### PR TITLE
Added an appdata.xml file for Linux software galleries

### PR DESCRIPTION
**Pull request by Mailaender**
See https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps for what this does. I took the descriptions from the official website. It also includes https://odrs.gnome.org/oars metadata.

### CODE REVIEW

File: distribution/linux/openrct2.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <p>
+      Original RollerCoaster Tycoon 2 game files are required in order to play OpenRCT2.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://camo.githubusercontent.com/888d993a9716208446bd0d5a762977d6b7993058/68747470733a2f2f692e696d6775722e636f6d2f6537434b3553632e706e67</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://openrct2.org/</url>
```
Comment (AaronVanGeffen) [inline distribution/linux/openrct2.appdata.xml]: It's a bit confusing, but the project's official website is https://openrct2.website/

Comment (Mailaender) [inline distribution/linux/openrct2.appdata.xml]: Changed.

File: distribution/linux/openrct2.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <screenshot type="default">
+      <image>https://camo.githubusercontent.com/888d993a9716208446bd0d5a762977d6b7993058/68747470733a2f2f692e696d6775722e636f6d2f6537434b3553632e706e67</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://openrct2.website/</url>
+  <url type="bugtracker">https://github.com/OpenRCT2/OpenRCT2/issues</url>
+  <url type="help">https://github.com/OpenRCT2/OpenRCT2/wiki</url>
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```
Comment (Gymnasiast) [inline distribution/linux/openrct2.appdata.xml]: I would argue it has some cartoon violence - drowning guests and exploding roller coaster cars (with guests) would count as such.

Comment (Mailaender) [inline distribution/linux/openrct2.appdata.xml]: Changed to `mild` which is
> Cartoon characters in unsafe situations

**Comment by Gymnasiast** [comment]
It looks good to me. After this is merged, we can have the translation team provide translations.
Could you give an example on how to add those? (You don't have to add them to this PR, just provide an example in the comments.)

**Comment by Mailaender** [comment]
https://github.com/hedgewars/hw/commit/b59494a3dd5f03086bbed6c7e4d5e47b7849ca82 is an example for manual appdata.xml translation.

File: distribution/linux/openrct2.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>openrct2.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>OpenRCT2</name>
+  <summary>An open-source re-implementation of RollerCoaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.</summary>
+  <description>
+    <p>
+      OpenRCT2 is an open-source re-implementation of RollerCoaster Tycoon 2 (RCT2). The gameplay revolves around building and maintaining an amusement park containing attractions, shops and facilities. The player must try to make a profit and maintain a good park reputation whilst keeping the guests happy. OpenRCT2 allows for both scenario and sandbox play. Scenarios require the player to complete a certain objective in a set time limit whilst sandbox allows the player to build a more flexible park with optionally no restrictions or finance.
```
Comment (janisozaur) [inline distribution/linux/openrct2.appdata.xml]: Can this be split into more lines in the file?

**Comment by janisozaur** [comment]
Other than overly long line, LGTM.

File: distribution/linux/openrct2.appdata.xml (hunk 3 above)
Comment (Mailaender) [inline distribution/linux/openrct2.appdata.xml]: Sure.

**Pull request merged**